src/promptflow/tests/sdk_cli_azure_test/e2etests/test_azure_cli_perf.py

```diff
@@ -6,18 +7,40 @@
 import pytest
 
 from promptflow._cli._user_agent import USER_AGENT as CLI_USER_AGENT  # noqa: E402
+from promptflow._sdk._telemetry import log_activity
 from promptflow._sdk._utils import ClientUserAgentUtil
+from sdk_cli_azure_test.recording_utilities import is_replay
 
 FLOWS_DIR = "./tests/test_configs/flows"
 DATAS_DIR = "./tests/test_configs/datas"
 
 
+def mock_log_activity(*args, **kwargs):
```

Comment:
seems `mock_log_activity` are same in two files? if so, how about make it a fixture and use it for these perf test?

Reply:
Of course, this is possible, but my understanding of fixtures is: is it a universal function? And not for certain test specific features?

Reply:
I won't make any modifications here because it's not possible to directly use fixtures in regular functions, and using them in other ways can actually increase maintenance costs.

Reply:
the function seems same, so even no fixture, how about keep it in one place, and import it in these two files?

Reply:
Yes, it can be done, and the next PR can be modified.